src/v3/obfuscate.ts

```diff
@@ -12,16 +12,15 @@ export const obfuscateData = (
   const salts = _data.proof.salts;
 
   // Obfuscate data by hashing them with the key
-  const obfuscatedData = fieldsToRemove
-    .filter(field => get(data, field))
-    .map(field => {
-      const value = get(data, field);
-      const salt = salts.find(s => s.path === field);
-      if (!salt) {
-        throw new Error(`Salt not found for ${field}`);
-      }
-      return toBuffer(`${salt.value}:${value}`).toString("hex");
-    });
+
+  const obfuscatedData = fieldsToRemove.map(field => {
```

Comment:
as I renamed the field in the schema I think we can do the same here => obfuscatedData=> obfuscated (same in the fn)